src/Nav.php

```diff
@@ -1,145 +1,202 @@
 <?php
-/**
- * @link http://www.yiiframework.com/
- * @copyright Copyright (c) 2008 Yii Software LLC
- * @license http://www.yiiframework.com/license/
- */
+declare(strict_types = 1);
 
 namespace Yiisoft\Yii\Bootstrap4;
 
-use yii\exceptions\InvalidConfigException;
 use Yiisoft\Arrays\ArrayHelper;
-use Yii;
+use Yiisoft\Yii\Bootstrap4\Exception\InvalidConfigException;
```

Comment:
That could be more general i.e. `InvalidConfigException` in view package widgets namespace.

Reply:
I will then create the exception in Yiisoft\Widget\Exception\InvalidConfigException,